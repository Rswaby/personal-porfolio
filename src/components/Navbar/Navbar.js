import React, { Component } from 'react'
import { Menu, Container } from 'semantic-ui-react'

export default class Navbar extends Component {
  state = { activeItem: 'Rohan' }
  render() {

    return (
      <div className="container">
        <Menu fixed="top" secondary inverted>
          <Container>
            <Menu.Item link as="a" href="/" name='Rohan' />
            <Menu.Item link as="a" href="/projects" name="projects" />
            <Menu.Menu position="right">
              <Menu.Item link as="a" href="/about" name="About" />
              <Menu.Item link as="a" href="/contact" name="Contact" />
            </Menu.Menu>
          </Container>
        </Menu>
      </div>
    )
  }
}
