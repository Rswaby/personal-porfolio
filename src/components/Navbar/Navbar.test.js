
/* eslint-disable */
import React from 'react';
import * as enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { mount } from 'enzyme';
import Navbar from './Navbar';
import renderer from 'react-test-renderer';

enzyme.configure({ adapter: new Adapter() });



describe('<Navbar /> ', () => {
    it('should render correctly', () => {
        const mock = jest.fn();
        const wrapper = mount(<Navbar />);
    });
    test('<Navbar/> snapshot', () => {
        ///
        const component = renderer.create(<Navbar />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();

    })

})
