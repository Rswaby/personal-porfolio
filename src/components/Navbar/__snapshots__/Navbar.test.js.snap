// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar />  <Navbar/> snapshot 1`] = `
<div>
  <div
    className="ui inverted secondary top fixed menu"
  >
    <div
      className="ui container"
    >
      <a
        className="link item"
        href="/"
        onClick={[Function]}
      >
        Rohan
      </a>
      <a
        className="link item"
        href="/projects"
        onClick={[Function]}
      >
        Projects
      </a>
      <div
        className="right menu"
      >
        <a
          className="link item"
          href="/About"
          onClick={[Function]}
        >
          About
        </a>
        <a
          className="link item"
          href="/Contact"
          onClick={[Function]}
        >
          Contact
        </a>
      </div>
    </div>
  </div>
</div>
`;
